module-ballerina-openapi/annotation.bal

```diff
@@ -30,6 +32,8 @@ public type ServiceInformation record {|
     string[]? excludeOperations = [];
     boolean failOnErrors = true;
     boolean embed = true;
+    string title = "";
+    string 'version = "";
```

Comment:
Shall we mark these optional ? I think that is the ballerina way of doing things :wink: 
```suggestion
    string title?;
    string 'version?;
```